### ISSUE TITLE
non_amd.html is broken in master

### ISSUE THREAD

**Issue by rlanvin** [jcbrand/converse.js#594]
Since the latest commits (which separates converse.js into different files), the file `non_amd.html` does not work anymore.

Replacing the last `<script>` by `converse-core.js` instead of `converse.js` helps a bit.

However, when trying to load MUC plugin, by adding another line at the end:
`<script type="text/javascript" src="src/converse-muc.js"></script>`

it breaks again and triggers the following error: 
> Uncaught TypeError: Cannot read property 'initialize' of undefined(anonymous function) @ non_amd.html:71

Basically `converse` object becomes undefined when adding converse-muc.

@jcbrand What is the procedure to load `converse-muc` in a non AMD environment with the new file structure?

**Comment by jcbrand** [comment]
I still need to figure that out again...
